File: BlackJack.ViewModels/Round/StartRoundView.cs
using System.Collections.Generic;

namespace BlackJack.ViewModels.Round
{
    public class StartRoundView
    {
        public string RoundResult { get; set; }
        public GamePlayerStartRoundViewItem Dealer { get; set; }
        public GamePlayerStartRoundViewItem Human { get; set; }
        public List<GamePlayerStartRoundViewItem> Bots { get; set; }
    }

    public class GamePlayerStartRoundViewItem
    {
        public string Name { get; set; }
        public int CardScore { get; set; }
        public List<string> Cards { get; set; }
    }
}

File: BlackJack.Angular/src/app/shared/models/start-round.view.ts
export class StartRoundView {
    roundResult: string;
    human: GamePlayerStartRoundViewItem;
    dealer: GamePlayerStartRoundViewItem;
    bots: GamePlayerStartRoundViewItem[];
}

export class GamePlayerStartRoundViewItem {
    name: string;
    cardScore: number;
    cards: string[];
}